en/index.md

```diff
@@ -1,13 +1,14 @@
 ---
+layout: page
```

Comment:
I think we want to remove `layout: page` here and change `lang: en` below to `redirect_from: /` like it was before.  This is a hiccup as a result of the big refactor 👍